utilities/examine_focus_w_grid_scipy.py

```diff
@@ -76,10 +76,9 @@ def variance_of_laplacian(image):
 rgb_weights = [0.299, 0.587, 0.114]
 
 # Initialize camera and grab a reference to the raw camera capture
-camera = PiCamera()
+camera = PiCamera(resolution=(640, 480),
```

Comment:
I didn't realize we'd been doing this at 640x480.  Is there a significant effect on performance if we do it at 1080p? Likely would be preferable to do it at higher res than 640x480.